crates/matrix-sdk-ui/src/timeline/controller/metadata.rs

```diff
@@ -75,6 +75,8 @@ pub(in crate::timeline) struct TimelineMetadata {
     pub aggregations: Aggregations,
 
     /// Given an event, what are all the events that are replies to it?
+    ///
+    /// Only works for remote events *and* replies which are remote-echoed.
```

Comment:
Should we rename `replies` to `remote_replies`? Not convinced, just asking.

Reply:
It's unclear in `remote_replies` which side is `remote`; in this case, it's both the original and the reply which must have been remote-echoed, and I think this information is more clearly described from the code comments + types used (only `OwnedEventId`, not `TimelineEventItemId` anywhere). If you don't mind too much, I'll keep this name.